product-service-ts/handlers/getProductsById.test.js

```diff
@@ -0,0 +1,33 @@
+import { getProductsById } from './getProductsById';
+import products from './products-mock.json';
+jest.mock('./products-mock.json');
+
+describe('getProductsById', () => {
+  afterEach(() => jest.resetAllMocks());
+  test('return product with existing id', async () => {
```

Comment:
each 'test' statement should start from 'should'

getProductsById  should return product with existing id

Reply:
Thanks, done.